File: IntroCPP/1_IntroProgramming/primes.cpp
#include <iostream>

bool isprime(int m);

int main(void) {

    //int counter = 0;
    int n = 100;
    for (int i = 1; i <= n; ++i){
        if (isprime(i)){
            std::cout << i << std::endl;
            //counter += 1;
        }
    }
    //std::cout << counter << std::endl;
}


bool isprime(int m){

    if (m < 2){
        return false;
    }
    for (int i = 2; i < m; ++i){
        if (m%i == 0){
            return false;
        }
    }
    return true;
}

File: IntroCPP/1_IntroProgramming/primes.py
def isprime(m):
    if m < 2:
        return False
    for i in range(2,m):
        if m%i == 0:
            return False
    return True

counter = 0
n = 100000

for i in range(2,n):
    if isprime(i):
        #print(i)
        counter += 1
        
print(counter)
